verifier/submission-verifier.js

```diff
@@ -41,12 +43,35 @@ class SubmissionActionVerifier {
         this.newStatus = this.actionMap.toStatus;
     }
 
+    isValidSubmitAction(role, aSubmission) {
```

Comment:
instead of using empty return statements can we flip this logic so that the error is thrown if the conditions are met else it naturally just returns